Document changes to Efficiency Correctors and running multiple times

See the discussion raised in #839.

If I try to follow these steps

cd docs
make clean
doxygen
make html
open _build/html/index.html

when I try to generate the html I get on the Melbourne cloud 
`sphinx-build -b html -d _build/doctrees   . _build/html
Making output directory...
Running Sphinx v0.6.6

`Exception occurred:
  File "/imports/home/fscutti/xAODAnaHelpers/docs/conf.py", line 83, in <module>
    import yaml
ImportError: No module named yaml
The full traceback has been saved in /scratch/sphinx-err-iMYne7.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-dev/>,
or report them in the tracker at <http://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [html] Error 1

I did not play too much with this but might be related with the sphinx version. Any idea?

--- a/docs/Algorithms.rst
+++ b/docs/Algorithms.rst
@@ -178,6 +178,27 @@ MuonCalibrator
 MuonEfficiencyCorrector
 ~~~~~~~~~~~~~~~~~~~~~~~
 
+Produces a container of muons decorated with efficiencies and scale factors.
+This container is not simply the one in input, but is a deepCopy of that. 
+This allows flexibility in decorating muons of systematically altered containers.
+The output container has a configurable name and is only created for MC events.
+When working with systematic uncertainties, a list of systematics is passed 
+to this algorithm *m_inputAlgoSystNames*\. This algorithm supports comma 
+separated lists as inputs, which will be considered as a unique list. The 
+systematic names in this list will be looked for to retrieve the muon containers 
+to decorate. Systematic variations on the decorations themselves might be 
+unnecessary for all systematic muon containers and might only be considered 
+for the nominal container (need of deepCopy). This is the default configuration. 
+Otherwise the  option *m_decorateWithNomOnInputSys*\ can be set to false. 
+The algorithm features the option *m_sysNamesForParCont*\ which is a list of systematic
+names. For each of them, a copy of the nominal muon container is put in the 
+store carrying the name of the systematic. These containers are only decorated
+with the nominal efficiencies and scale factors. The use case of this are MET
+systematics for which one does not want systematic variations on efficiencies,
+but still wants to retrieve a nominal muon container in the tree algo. Retrieving
+the nominal would indeed carry all the unnecessary uncertainties.
+
+
 MuonSelector
 ~~~~~~~~~~~~
 
@@ -192,6 +213,8 @@ ElectronCalibrator
 
 ElectronEfficiencyCorrector
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+See the MuonEfficiencyCorrector
 
 ElectronSelector
 ~~~~~~~~~~~~~~~~

